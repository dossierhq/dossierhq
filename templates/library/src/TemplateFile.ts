import { Temporal } from '@js-temporal/polyfill';

export interface TemplateInterface {
  value: string;
}

export const TEMPLATE_VALUE: TemplateInterface = {
  value: 'hello world',
};

export const AN_INSTANT = Temporal.Instant.from('2022-04-30T07:51:25.56Z');
