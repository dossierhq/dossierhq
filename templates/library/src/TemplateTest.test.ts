import { describe, expect, test } from 'vitest';
import { TEMPLATE_VALUE } from './TemplateFile.js';

describe('hello', () => {
  test('world', () => {
    expect(TEMPLATE_VALUE.value).toBe('hello world');
  });

  test('inline snapshot', () => {
    expect({ what: 123 }).toMatchInlineSnapshot(`
      {
        "what": 123,
      }
    `);
  });

  test('snapshot', () => {
    expect({ what: 123 }).toMatchSnapshot();
  });
});
