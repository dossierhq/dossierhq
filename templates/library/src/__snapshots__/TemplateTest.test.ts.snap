// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`hello > snapshot 1`] = `
{
  "what": 123,
}
`;
